docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml

```diff
@@ -155,12 +155,61 @@ test = script do
   -- This transfer happened, so balance is now 900
   ((_, alice_account)::_) <- query @Account alice
   alice_account.amount === 900.0
+  -- and there's an Iou from Alice's bank to the shop
+  [(_, iou_bank)] <- query @Iou bank
+  iou_bank === Iou with issuer = bank, owner = shopOwner, amount = 100.0
+  -- Now let's try to go over the limit
   submit alice $ exerciseCmd shop (OrderItemTrustedLimited alice)
-  -- This one didn't, so balance is still 900
+  -- This one doesn't pass, so balance is still 900
   ((_, alice_account)::_) <- query @Account alice
   alice_account.amount === 900.0
+  -- and the newly created Iou is in Alice's name instead of her bank's
+  [(_, iou_alice)] <- query @Iou alice
+  [(_, iou_bank)] <- query @Iou bank
+  iou_bank === Iou with issuer = bank, owner = shopOwner, amount = 100.0
+  iou_alice === Iou with issuer = alice, owner = shopOwner, amount = 100.0
 
-  pure ()
+test2 = script do
+  bank <- allocateParty "Bank"
+  alice <- allocateParty "Alice"
+  shopOwner <- allocateParty "ShopOwner"
+
+  alice_account_id <-
+    submitMulti [bank, alice] [] $ createCmd Account with
+      issuer = bank
+      owner = alice
+      amount = 1000.0
+
+  submitMulti [alice, shopOwner] [] $
+    createAndExerciseCmd
+      (TestThrewTransferLimitExceeded with
+        owner = alice
+        newOwner = shopOwner)
+      (TestThrewTransferLimitExceededChoice with
+        accountId = alice_account_id)
```

Comment:
Should we have snake case here? Is this a practice I'm not aware of?

Reply:
I'm at fault here I believe. Generally speaking we follow the Haskell convention of camelCase everywhere, but I find snake_case so much more readable that I tend to forget.

Reply:
ah yes I was going for consistency with Gary's tests. I'll switch it all to camelCase now